tests/chainerx_tests/unit_tests/routines_tests/test_manipulation.py

```diff
@@ -181,6 +181,40 @@ def test_rollaxis_invalid(in_shape, axis, start):
         else:
             chainerx.rollaxis(a, axis, start)
 
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize_pytest('shape,pad_width,mode,constant_values', [
+    ((1,), 1, "constant", 0),
```

Comment:
Use single quotes for string literals for consistency.

Reply:
Done.